### PR TITLE
7zr: add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by andrik**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages.pt-BR/common/7zr.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 7zr
+
+> Um compactador de arquivos com alta taxa de compressão.
+> Versão do `7z` com suporte apenas para o formato 7z.
```
Comment (waldyrious) [inline pages.pt-BR/common/7zr.md]: Talvez "para o formato `.7z`" poderia ficar mais claro -- o que achas, @andrik?

Comment (andrik) [reply pages.pt-BR/common/7zr.md]: boa

Comment (waldyrious) [reply pages.pt-BR/common/7zr.md]: Eu manteria os backticks à volta da extensão. Podes inclui-los?

Comment (andrik) [reply pages.pt-BR/common/7zr.md]: Claro

**Pull request merged**